reference/shaders-msl-no-opt/asm/frag/texture-access.swizzle.asm.frag

```diff
@@ -0,0 +1,157 @@
+#pragma clang diagnostic ignored "-Wmissing-prototypes"
+
+#include <metal_stdlib>
+#include <simd/simd.h>
+
+using namespace metal;
+
+// Returns 2D texture coords corresponding to 1D texel buffer coords
+uint2 spvTexelBufferCoord(uint tc)
+{
+    return uint2(tc % 4096, tc / 4096);
+}
+
+enum class spvSwizzle : uint
+{
+    none = 0,
+    zero,
+    one,
+    red,
+    green,
+    blue,
+    alpha
+} ;
```

Comment:
Space between } and ;

Reply:
I don't know how to get rid of that. It's generated by the `end_scope_decl()` function, which assumes its string argument is non-empty.

Reply:
Ok, should be trivial. I can fix that later.

Reply:
Oh, there's a variant that doesn't take an argument and which doesn't put the space in. Silly me.